lib/components/learn-more/index.tsx

```diff
@@ -0,0 +1,37 @@
+import { Image, Heading, Button, Grid } from '@chakra-ui/react';
+import Link from 'next/link';
+import React from 'react';
+
+interface LearnMoreProps {
+  imgSrc: string;
+  text: string;
+}
+
+const LearnMore: React.FC<LearnMoreProps> = ({ imgSrc, text }) => {
+  return (
+    <div data-cy="learn-more-section" className="about-us-image-container">
+      <Image
+        width={'100%'}
+        height={'calc(100vh - 100px)'}
+        objectFit="cover"
+        alt="Learn More Image"
+        src={imgSrc}
+      />
+      <Grid gap={'1rem'} justifyItems="center" className="about-us-caption-button">
```

Comment:
All CSS that was put in globals.css can be duplicated by using the Flex component and using width, maxWidth, margin, and padding.  Play around in https://chakra-ui.com/docs/components/form/button